pkg/client/defaults.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2018 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+import (
+	"github.com/heptio/sonobuoy/pkg/config"
+)
+
+// NewGenConfigWithDefaults is a GenConfig using the default config and Conformance mode
+func NewGenConfigWithDefaults() *GenConfig {
```

Comment:
I think it'd be ok to drop the "WithDefaults". I'd assume NewGenConfig() would have defaults embedded in it implicitly, but I'm also fine with NewGenConfigWithDefaults.

Reply:
I added the WithDefaults() to align with NewConfigWithDefaults(), but I can drop this